includes/class-wpfaevent-i18n.php

```diff
@@ -3,45 +3,23 @@
 /**
  * Define the internationalization functionality
  *
- * Loads and defines the internationalization files for this plugin
- * so that it is ready for translation.
- *
  * @link       https://fossasia.org
  * @since      1.0.0
  *
- * @package    Wpfaevent
- * @subpackage Wpfaevent/includes
+ * @package    WPFA_Event
+ * @subpackage WPFA_Event/includes
  */
 
-/**
- * Define the internationalization functionality.
- *
- * Loads and defines the internationalization files for this plugin
- * so that it is ready for translation.
- *
- * @since      1.0.0
- * @package    Wpfaevent
- * @subpackage Wpfaevent/includes
- * @author     FOSSASIA <contact@fossasia.org>
- */
```

Comment:
Why do these comments need to be deleted? This is not executed, but it could help in future development. So, what is the goal?